web_widget_tolocalestring/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Web Widget toLocaleString
+==============
+
+This module add 'tolocalestring' widget in tree view for formatting numbers or not to display the values 0 : 
```

Comment:
This module adds the possibility to use 'tolocalestring' widget in tree view, in order to improve numbers  formatting or to not display the 0 values: